### DISCUSSION TITLE
Desktop Application

### DISCUSSION THREAD

**Discussion by Marcii0** [benawad/dogehouse#1756] [Q&A] [answered]
Perhaps, a Desktop Application you could download, for example: Discord, Messenger, etc..
Don't know if this has been suggested before

**Comment by babeuh** [comment]
A desktop app for Dogehouse built with electron is available for Windows, Mac, and Linux

Download links are in the releases section on github.

There is also a list of unofficial clients: #988

If you meant something else, please clarify.

**Comment by amitojsingh366** [comment, marked as answer]
https://dogehouse.tv/download
